homeassistant/components/camera/__init__.py

```diff
@@ -402,6 +401,8 @@ def unsub_track_time_interval(_event: Event) -> None:
     )
 
     async def get_ice_server() -> RTCIceServer:
```

Comment:
Can we let this function return a list instead of a single item?

Reply:
Yes, we can. You can change it in this PR

Reply:
Done in PR https://github.com/home-assistant/core/pull/129024